.github/workflows/wheels.yml

```diff
@@ -139,17 +139,15 @@ jobs:
         if: contains(matrix.os, 'macos')
         # Install coreutils which includes `nproc` used by `make -j` in suitesparse.sh
         #
-        # GitHub actions comes with libomp already installed, but for its native arch only. Must build universal one
-        # manually so that both x86 and arm builds can be built.
+        # Explicitly install libomp to be clear about the dependency.
+        #
+        # libomp determines the minimum macOS version that we can build for
         run: |
           brew fetch --retry coreutils && brew install coreutils
-          brew fetch --retry libomp && brew install libomp
-          if [[ ${{ matrix.cibw_archs }} == "arm64" ]] ; then
-            echo "Building universal libomp manually"
-            sh add_arm_to_libomp_dylib.sh || exit 1
-          fi
+          brew fetch --retry libomp && brew reinstall libomp
+          echo MACOSX_DEPLOYMENT_TARGET=$(otool -l $(brew --prefix libomp)/lib/libomp.dylib | grep minos | awk '{print $2}') >> $GITHUB_ENV
 
-      - uses: pypa/cibuildwheel@v2.16
+      - uses: pypa/cibuildwheel@v2.21.3
```

Comment:
If a point release is specified then dependabot will create update PRs for every new point release. We probably want to stick with `2.21` instead. That will pull in the latest point release.

Reply:
Done, thanks